### DISCUSSION TITLE
[Skia] Uno XAML Hot Reload Fail with Error ENC0003

### DISCUSSION THREAD

**Discussion by lindexi** [unoplatform/uno#15720] [Q&A] [answered]
Uno XAML Hot Reload Fail in Skia platform with Error ENC0003, the detail fail message is:

```
Uno.UI.SourceGenerators\Uno.UI.SourceGenerators.XamlGenerator.XamlCodeGenerator\LocalizationResources.cs (Line 2): Error ENC0003: Adding 'attribute' requires restarting the application
```

It is useless after adding `<GenerateAssemblyInfo Condition="'$(Configuration)'=='Debug'">false</GenerateAssemblyInfo>` to `Directory.Build.props` or the App.Skia.Wpf csproj

- My Uno version : 5.1.75
- My VisualStudio Version: VisualStudio 2022 17.9.0
- My dotnet version: 8.0.200

**Comment by jeromelaban** [comment, marked as answer]
Thanks, this is an issue from the .NET for which various workarounds are mentioned: https://github.com/dotnet/sdk/issues/36666

**Reply by lindexi** [reply]
Thank you very much. @jeromelaban
